Source/Csla.Data.SqlClient/Csla.Data.SqlClient.csproj

```diff
@@ -26,6 +26,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Azure.Identity" Version="1.12.0" />
```

Comment:
Why is this necessary?

Reply:
This was necessary to resolve a package vulnerability warning - I forced it to use the non-vulnerable version.